### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nikhil-ghate**

### CODE REVIEW

**Comment by sanjeevyadavIT** [comment]
Hello @nikhil-ghate Thank you for creating the PR, could you rephrase the question something like this and also resolve merge conflict

> What certain qualities in the candidate you look for that aligns with your company values?

**Comment by nikhil-ghate** [comment]
| 25 | What certain qualities in the candidate you look for that aligns with your company values? And which are the qualities that I have from that?

**Comment by sanjeevyadavIT** [comment]
I don't understand why would I ask `And which are the qualities that I have from that?` to a person who is interviewing me. You could ask a question something similar to it(like feedback) when you have got the job offer, but when I am being interviewed this part doesn't make sense

**Comment by nikhil-ghate** [comment]
Okay.

What about 1st que

**Comment by sanjeevyadavIT** [comment]
Yes first part is ok, please go ahead, and edit question & remove merge conflict.

File: README.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ At the time of joining, the HR portrays the company like it is match made in Hea
 | 22 | Do you have a higher turnover rate for minorities in the company as a whole, and in the department specifically?|
 | 23 | If there is downtime during work hours, what are the expectations? (aka: can I read and study for a cert during work hours if there's nothing to do?)|
 | 24 | Who will I be working most closely with?
+| 25 | According to you what are the qualities that new employee should must have? And which are the qualities that I have from that?
```
Comment (nikhil-ghate) [inline README.md:34]: | 25 | What certain qualities in the candidate you look for that aligns with your company values ?

**Review by nikhil-ghate — COMMENTED** [review]
Go ahead
